centreon/www/front_src/src/ApiTokens/TokenCreation/Form.tsx

```diff
@@ -147,6 +150,7 @@ const FormCreation = ({ data, isMutating }: Props): JSX.Element => {
           anchorElDuration={{ anchorEl, setAnchorEl }}
           openPicker={{ open, setOpen }}
           setIsDisplayingDateTimePicker={setIsDisplayingDateTimePicker}
+          windowHeight={height}
```

Comment:
Why do you need the height of the window?

Reply:
To set the calendar's height to approximately 1/3 of the user's screen, ensuring visibility regardless of screen resolutions. This adjustment allows the calendar's height to adapt to new screen dimensions and also adjusts the scrollbar accordingly when the screen dimensions change :tropical_fish: